scripts/python/SLiM_sims.py

```diff
@@ -26,6 +26,26 @@ def run_slim(N, bot, outpath, slim_path):
     print(err)  # prints error message
 
 
+def bgzip(outpath):
+
+    # Find all VCFs in outpath
+    process_find = subprocess.Popen(['find', outpath, '-name', '*.vcf'],
+                                    stdout=subprocess.PIPE,
+                                    stderr=subprocess.PIPE,
+                                    universal_newlines=True)
+
+    # bgzip all found VCFs
+    process_bgzip = subprocess.Popen(['xargs', '-n1', 'bgzip', '-f'],
```

Comment:
Thought - do we need to tabix these VCFs as well post-bgzipping? That could be included in `bgzip()` after the bgzipping but careful to glob for `vcf.gz` instead of `vcf`

Reply:
Yeah good thought! I don't remember if we need to index them or not. I'll know soon when I go to run LDhat so I say we leave this for now and open another PR with that change when the time comes.